spec/nodes/addons_spec.rb

```diff
@@ -60,9 +60,11 @@ def addons(input)
       example { expect(addons(ssh_known_hosts: ['git.example.org', 'git.example.com']).ssh_known_hosts).to be == ['git.example.org', 'git.example.com'] }
     end
 
-    context 'apt_packages' do
-      example { expect(addons(apt_packages: 'curl').apt_packages).to be == ['curl'] }
-      example { expect(addons(apt_packages: ['curl', 'git']).apt_packages).to be == ['curl', 'git'] }
+    context 'apt' do
+      example { expect(addons(apt: { 'sources': 'deadsnakes'}).apt.sources).to be == ['deadsnakes'] }
```

Comment:
[This fails the build](https://travis-ci.org/travis-ci/travis-yaml/jobs/67432591).

Reply:
Old Ruby version running on travis maybe? Looks like its failing on the newer : syntax. I'd say either upgrade the ruby version used on travis or change the hashes to the old rocket style 'something' => 'deadsnakes'.